addons/hr_contract/migrations/11.0.1.0/openupgrade_analysis_work.txt

```diff
@@ -0,0 +1,28 @@
+---Fields in module 'hr_contract'---
+hr_contract  / hr.contract              / company_id (many2one)         : NEW relation: res.company
+hr_contract  / hr.contract              / resource_calendar_id (many2one): NEW relation: resource.calendar
+# Nothing to do, new relations
+
+hr_contract  / hr.contract              / state (selection)             : selection_keys is now '['cancel', 'close', 'draft', 'open', 'pending']' ('['close', 'draft', 'open', 'pending']')
+# Nothing to do, nothing related
+
+hr_contract  / hr.contract              / trial_date_start (date)       : DEL
+# Nothing to do, nothing related
+
+hr_contract  / hr.contract              / website_message_ids (one2many): NEW relation: mail.message
+# Nothing to do, new relations
+
+hr_contract  / hr.contract              / working_hours (many2one)      : DEL relation: resource.calendar
```

Comment:
This is clearly a rename of the field from `working_hours` to `resource_calendar_id`, and more if you check the label from both fields in v10 and v11: `Working Schedule`. Please pay attention to these details.